cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -1203,10 +1206,17 @@ private static void writeLimitsGroup(Identifiable<?> identifiable, OperationalLi
         // Write the OperationalLimitSet
         String operationalLimitSetId;
         String operationalLimitSetName;
-        String propertyKey = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.OPERATIONAL_LIMIT_SET + "_" + limitsGroup.getId();
+        String propertyKey = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.OPERATIONAL_LIMIT_SET;
         if (identifiable.hasProperty(propertyKey)) {
             operationalLimitSetId = limitsGroup.getId();
-            operationalLimitSetName = identifiable.getProperty(propertyKey);
+            try {
+                ObjectMapper mapper = new ObjectMapper();
+                JsonNode propertyNode = mapper.readTree(identifiable.getProperty(propertyKey));
```

Comment:
Should the check on `identifiable.hasProperty(propertyKey)` be now equivalent to the test on `identifiable.hasProperty(propertyKey)` + `null != propertyNode && null != limitsGroupNode` or equivalent? If so the else condition should include also this case?  
Maybe we should also include the cases when `propertyNode` is null to avoid NPE exception (need to really validate if that exception is thrown if the JsonNode is not found)